lib/widgets/subscription_list.dart

```diff
@@ -256,6 +263,9 @@ class SubscriptionItem extends StatelessWidget {
                 count: unreadCount,
                 backgroundColor: swatch,
                 bold: true)),
+          ] else if (hasAllUnreadsMuted && subscription.isMuted) ...[
+            const SizedBox(width: 12),
+            const MutedUnreadBadge(),
```

Comment:
I _think_ TODO above this also applies here:

```dart
// TODO(#747) show @-mention indicator when it applies
```

In the web app, if I have an unread mention in a muted stream, where the topic is not unmuted, I see a dot and an @-mention indicator:

<img width="255" alt="image" src="https://github.com/user-attachments/assets/43a8c515-86a1-4791-8397-c2f25be782b6">

So let's copy that TODO into this case too.